packages/joint-core/src/util/util.mjs

```diff
@@ -1086,10 +1076,9 @@ export const getElementBBox = function(el) {
 // See http://james.padolsey.com/javascript/sorting-elements-with-jquery/.
 export const sortElements = function(elements, comparator) {
 
-    var $elements = $(elements);
-    var placements = $elements.map(function() {
+    elements = $(elements).toArray();
+    var placements = elements.map(function(sortElement) {
 
-        var sortElement = this;
```

Comment:
We are relying on `this` being `sortElement` later ([here](https://github.com/clientIO/joint/pull/2414/files#diff-065c48be1d93599716d40fab9433aec775a3e3233769985f739384665dfbc0caR1090) and [here](https://github.com/clientIO/joint/pull/2414/files#diff-065c48be1d93599716d40fab9433aec775a3e3233769985f739384665dfbc0caR1095), but the new code doesn't reassign `this` as expected, right?